### PR TITLE
feat: artwork page - link to user idv flow if applicable

### PR DESCRIPTION
**Pull request by erikdstock**
This PR adds the link-to-identity-verification functionality similar to #6342 to our artwork pages, prompting a user with a pending identity verification to complete it. The affected component contains logic for several different sale types but i have only added a test for one of them - something probably worth addressing. There is also some basically duplicated logic between this and Registration.tsx, which is for the sale page and does not use relay.

image

### CODE REVIEW

File: src/v2/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarBidAction.tsx [hunk 1]
```diff
@@ -128,11 +135,14 @@ export class ArtworkSidebarBidAction extends React.Component<
...
-      bidder: sale.registrationStatus,
-    })
+    const userLacksIdentityVerification =
+      sale.requireIdentityVerification && !me?.identityVerified
+    const pendingIdentityVerification = me?.pendingIdentityVerification
+
+    const shouldPromptIdVerification =
+      !qualifiedForBidding &&
+      userLacksIdentityVerification &&
+      Boolean(pendingIdentityVerification)
```
Comment (erikdstock) [inline src/v2/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarBidAction.tsx:145]: Maybe the function i have been removing should actually be updated to return an interface full of idv-related properties like this instead.

Comment (bhoggard) [reply src/v2/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarBidAction.tsx:145]: I think this is fine, as long as the login remains confined to this component.

**Review by bhoggard — APPROVED** [review]

**Comment by admbtlr** [comment]
Thanks for using semantic style of the PR title! 🙌 Strictly speaking, there isn't an `enhancement` option, so maybe just go with `feat`?

**Comment by erikdstock** [comment]
@bhoggard, this is now complete and ready for squash/merge.

**Comment by erikdstock** [comment]
#squashongreen

**Pull request merged**